vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion.cc

```diff
@@ -119,6 +119,23 @@ void Promotion::Fetch(ledger::FetchPromotionCallback callback) {
     return;
   }
 
+  // If we fetched promotions recently, fulfill this request from the
+  // database instead of querying the server again
+  if (!ledger::is_testing) {
+    const uint64_t last_promo_stamp =
+        ledger_->GetUint64State(ledger::kStatePromotionLastFetchStamp);
+    const uint64_t now = braveledger_time_util::GetCurrentTimeStamp();
+    if (now - last_promo_stamp < 10 * base::Time::kSecondsPerMinute) {
```

Comment:
please move `10 * base::Time::kSecondsPerMinute` to the top of the file as const, so that it's easier to sot

Reply:
Done.